tests/test_settings.py

```diff
@@ -477,7 +478,12 @@ def test_settings__missing_config_file__uses_defaults_and_warns(tmp_path, caplog
     settings = Settings.config(config_file=missing_file)()
     assert settings.dict() == make_settings_dict()
     assert "Failed to load configuration file:" in caplog.text
-    assert str(missing_file) in caplog.text
+    # On Windows path escape sequences are doubled in repr()
+    assert (
+        repr(str(missing_file))
+        if platform.system() == "Windows"
+        else str(missing_file) in caplog.text
+    )
```

Comment:
It's hard to see that the `... in caplog.text` on the last line also applies to the Windows-side of the if-else condition. What about this?

```suggestion
    missing_file_str = str(missing_file)
    # On Windows path escape sequences are doubled in repr()
    if sys.platform.startswith("win"):
        missing_file_str = repr(missing_file_str)
    assert missing_file_str in caplog.text
```